src/main/java/org/zalando/springframework/web/logging/DefaultHttpLogger.java

```diff
@@ -28,7 +28,7 @@
 import java.util.List;
 import java.util.Map;
 
-public class DefaultHttpLogger implements HttpLogger {
+public final class DefaultHttpLogger implements HttpLogger {
```

Comment:
> Why final?

Because it's not intended to be subclassed, is it? If somebody needs stuff from it he/she is free to use composition.